<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="6" passed="4" failed="2" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2025-04-13T14:01:08 IST" name="Default suite" finished-at="2025-04-13T14:01:20 IST" duration-ms="12419">
    <groups>
    </groups>
    <test started-at="2025-04-13T14:01:08 IST" name="Default test" finished-at="2025-04-13T14:01:20 IST" duration-ms="12419">
      <class name="com.api.testcases.DataDrivenTest">
        <test-method signature="testCreateUser(java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String)[pri:1, instance:com.api.testcases.DataDrivenTest@6aeb35e6]" started-at="2025-04-13T14:01:10 IST" name="testCreateUser" data-provider="userData" finished-at="2025-04-13T14:01:13 IST" duration-ms="3630" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[2020]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[user1]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[chintu]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[kumar]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[aa@gmail.com]]>
              </value>
            </param>
            <param index="5">
              <value>
                <![CDATA[test@1]]>
              </value>
            </param>
            <param index="6">
              <value>
                <![CDATA[1234567891]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testCreateUser -->
        <test-method signature="testCreateUser(java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String)[pri:1, instance:com.api.testcases.DataDrivenTest@6aeb35e6]" started-at="2025-04-13T14:01:13 IST" name="testCreateUser" data-provider="userData" finished-at="2025-04-13T14:01:14 IST" duration-ms="1328" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[2021]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[user2]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[raju]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[gupta]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[bb@gmail.com]]>
              </value>
            </param>
            <param index="5">
              <value>
                <![CDATA[test@2]]>
              </value>
            </param>
            <param index="6">
              <value>
                <![CDATA[1234567892]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testCreateUser -->
        <test-method signature="testCreateUser(java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String)[pri:1, instance:com.api.testcases.DataDrivenTest@6aeb35e6]" started-at="2025-04-13T14:01:14 IST" name="testCreateUser" data-provider="userData" finished-at="2025-04-13T14:01:16 IST" duration-ms="1367" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[2022]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[user3]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[bheem]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[pathak]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[cc@gmail.com]]>
              </value>
            </param>
            <param index="5">
              <value>
                <![CDATA[test@3]]>
              </value>
            </param>
            <param index="6">
              <value>
                <![CDATA[1234567893]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testCreateUser -->
        <test-method signature="testDeleteUser(java.lang.String)[pri:1, instance:com.api.testcases.DataDrivenTest@6aeb35e6]" started-at="2025-04-13T14:01:16 IST" name="testDeleteUser" data-provider="username" finished-at="2025-04-13T14:01:17 IST" duration-ms="1406" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[user1]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [200] but found [404]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [200] but found [404]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1590)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:1431)
at org.testng.Assert.assertEquals(Assert.java:1395)
at org.testng.Assert.assertEquals(Assert.java:1441)
at com.api.testcases.DataDrivenTest.testDeleteUser(DataDrivenTest.java:39)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:687)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:995)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:741)
at org.testng.TestRunner.run(TestRunner.java:616)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testDeleteUser -->
        <test-method signature="testDeleteUser(java.lang.String)[pri:1, instance:com.api.testcases.DataDrivenTest@6aeb35e6]" started-at="2025-04-13T14:01:17 IST" name="testDeleteUser" data-provider="username" finished-at="2025-04-13T14:01:19 IST" duration-ms="1380" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[user2]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [200] but found [404]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [200] but found [404]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1590)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:1431)
at org.testng.Assert.assertEquals(Assert.java:1395)
at org.testng.Assert.assertEquals(Assert.java:1441)
at com.api.testcases.DataDrivenTest.testDeleteUser(DataDrivenTest.java:39)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:687)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:995)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:741)
at org.testng.TestRunner.run(TestRunner.java:616)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testDeleteUser -->
        <test-method signature="testDeleteUser(java.lang.String)[pri:1, instance:com.api.testcases.DataDrivenTest@6aeb35e6]" started-at="2025-04-13T14:01:19 IST" name="testDeleteUser" data-provider="username" finished-at="2025-04-13T14:01:20 IST" duration-ms="1352" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[user3]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testDeleteUser -->
      </class> <!-- com.api.testcases.DataDrivenTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
